wiki/Main_Page/pt-br.md

```diff
@@ -1,61 +1,87 @@
 ---
 layout: main_page
-outdated: true
-outdated_since: 60193a8c102db30d5445e004fecc2ce28f073b60
 ---
 
 <div class="wiki-main-page__blurb">
 Bem-vindo ao osu! wiki, um projeto que contém uma ampla variedade de informação relacionada a osu!.
 </div>
 
-
 <div class="wiki-main-page__panels">
-<div class="wiki-main-page-panel">
-
+<div class="wiki-main-page-panel wiki-main-page-panel--full">
 
 # Introdução
 
-[Bem-vindo](/wiki/Welcome) • [Registro](/wiki/Registration) • [Instalação](/wiki/Installation) • [Interface](/wiki/Interface) • [Opções do Jogo](/wiki/Options) • [Teclas de Atalho](/wiki/Shortcut_key_reference) • [Mapa do Site](/wiki/Sitemap) • [Guias](/wiki/Guides) • [Central de Ajuda](/wiki/Help_Centre)
+[Bem-vindo](/wiki/Welcome) • [Instalação](/wiki/Installation) • [Registro](/wiki/Registration) • [Central de Ajuda](/wiki/Help_Centre) • [Perguntas Frequentes](/wiki/FAQ)
 
 </div>
 <div class="wiki-main-page-panel">
 
+# Cliente do jogo
 
-# Jogabilidade
+[Interface](/wiki/Interface) • [Opções](/wiki/Options) • [Opções visuais](/wiki/Visual_Settings) • [Teclas de Atalho](/wiki/Shortcut_key_reference) • [Arquivo de configuração](/wiki/osu!_Program_Files/User_Configuration_File) • [Arquivos do programa](/wiki/osu!_Program_Files)
```

Comment:
```suggestion
[Interface](/wiki/Interface) • [Opções](/wiki/Options) • [Configurações visuais](/wiki/Visual_Settings) • [Referência de teclas de atalho](/wiki/Shortcut_key_reference) • [Arquivo de configuração](/wiki/osu!_Program_Files/User_Configuration_File) • [Arquivos do programa](/wiki/osu!_Program_Files)
```
Consistency with currently translated shortcut key reference and visual settings articles, and it is also closer to English version.
If you feel like it's a mouthful, I could open up a PR to update the aforementioned article(s) to bear that title instead. `Configurações visuais` wasn't translated by me but I believe we could apply the changes to that one too if needed, to be closer with `/wiki/Options` (whenever that gets translated)

`Arquivos do programa` could have `de` instead but since it's refering to osu!'s program files I believe this is totally fine. No need to change it.